zebra-test/src/mock_service.rs

```diff
@@ -959,6 +959,7 @@ impl<Request, Response, Error> ResponseSender<Request, Response, Error> {
     }
 }
 
+#[allow(dead_code)]
```

Comment:
Optional/Nitpick: Do we want to open an issue for adding this as a constraint on the `MockService` struct and its methods? I think it's also fine to leave it as dead code for now, it's still useful for documentation.

Reply:
let just leave it, i don't think is very important.